app/main.py

```diff
@@ -39,3 +39,4 @@ def get_application() -> FastAPI:
 
 
 app = get_application()
+app.mount("/static/css", StaticFiles(directory="static/css"), name="css")
```

Comment:
Shouldn't we just mount the `/static` directory as a whole? Why narrow it down to `/static/css`?